apps/pollbook/frontend/src/api.ts

```diff
@@ -64,7 +64,7 @@ export const getPollbookConfigurationInformation = {
       this.queryKey(),
       () => apiClient.getPollbookConfigurationInformation(),
       {
-        refetchInterval: 1000,
+        refetchInterval: DEFAULT_QUERY_REFETCH_INTERVAL,
```

Comment:
the effect here is that the network polling and the statistics have the same refetch interval, correct? if the performance impact is negligible, seems fine, but I think we could do with a larger interval for statistics (5 or even 10 seconds). I'm not sure how the `getVoter` endpoint is being used to know what's appropriate there.

Reply:
the event polling between computers is more frequent: https://github.com/votingworks/vxsuite/blob/main/apps/pollbook/backend/src/globals.ts#L37